packages/app/src/contacts/components/contact-list/contact-list.component.tsx

```diff
@@ -145,6 +145,7 @@ const ContactList: FunctionComponent<Props> = ({
   const tableRef = createRef<HTMLDivElement>()
   const CategoryLabels = styled(Labels)`
     align-items: end;
+    background-color: var(--rowBackground) !important;
```

Comment:
`!important` it's an exception, is it really an exception? 🤔

Reply:
To change background color here I need to override "!import" in another place, so it needs to stay